homeassistant/components/mqtt/cover.py

```diff
@@ -129,15 +129,15 @@ def validate_options(value):
     ):
         _LOGGER.warning(
             "using 'value_template' for 'position_topic' is deprecated "
-            "and will be removed from Home Assistant in version 2021.6"
-            "please replace it with 'position_template'"
+            "and will be removed from Home Assistant in version 2021.6, "
+            "please replace it with 'position_template'."
```

Comment:
We don't end logging messages with period.

Reply:
Removed